backend/src/main/java/de/echochamber/backend/controller/Mapper.java

```diff
@@ -0,0 +1,119 @@
+package de.echochamber.backend.controller;
+
+import de.echochamber.backend.api.Answer;
+import de.echochamber.backend.api.Poll;
+import de.echochamber.backend.api.PossibleAnswer;
+import de.echochamber.backend.api.User;
+import de.echochamber.backend.model.AnswerEntity;
+import de.echochamber.backend.model.PollEntity;
+import de.echochamber.backend.model.PossibleAnswerEntity;
+import de.echochamber.backend.model.UserEntity;
+import de.echochamber.backend.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.persistence.EntityNotFoundException;
+import java.util.*;
+
+@Service
+public class Mapper {
+
+    private final UserService userService;
+
+    @Autowired
+    public Mapper(UserService userService) {
+        this.userService = userService;
+    }
```

Comment:
```suggestion
```

This violates SOC, separation of concerns. The mapper should only map between objects and not perform lookups. See comment below.